router/route.go

```diff
@@ -0,0 +1,140 @@
+package router
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+var (
+	// DefaultLocalMetric is default route cost for local network
+	DefaultLocalMetric = 1
+	// DefaultNetworkMetric is default route cost for micro network
+	DefaultNetworkMetric = 10
+)
+
+// AddPolicy defines routing table addition policy
+type AddPolicy int
```

Comment:
Remove AddPolicy. Or change the Policy to be a function that takes a Route and allows the caller to define how to handle it e.g like Retry and Backoff Func in client https://github.com/micro/go-micro/blob/master/client/options.go#L51 then provide a default like AlwaysOverride